ledger-tool/src/main.rs

```diff
@@ -2710,6 +2813,16 @@ fn main() {
                                 bank.hash(),
                                 full_snapshot_archive_info.path().display(),
                             );
+
+                            if is_minimize {
+                                let starting_epoch = bank.epoch_schedule().get_epoch(snapshot_slot);
+                                let ending_epoch =
+                                    bank.epoch_schedule().get_epoch(ending_slot.unwrap());
+                                if starting_epoch != ending_epoch {
+                                    warn!("Minimized snapshot range crosses epoch boundary ({} to {}). Bank hashes after {} will not match replays from a full snapshot",
```

Comment:
Added a warning here. Bank hashes will differ after an epoch boundary due to inflation rewards being distributed. Inflation is dependent on total capitalizatiom, which changes when we remove accounts.